### DISCUSSION TITLE
AsyncLogging::threadFunc() 中判断buffersToWrite.size()大于25和下面删除只剩2个不一致

### DISCUSSION THREAD

**Discussion by zhangke96** [chenshuo/muduo#592] [General]
if (buffersToWrite.size() > 25) 这里判断要写的日志buffer数量大于25，下面代码删除到只剩2个，是有意这么设计的吗？

**Comment by qzheng130** [comment]
理解的不一定对，快速产生大量的日志，往往只有前面的日志对分析问题有帮助，所以删掉了后面的。
